client/src/components/Event/Event.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from "react";
```

Comment:
Tady ti krici TypeScript, ze to neni pouzite. Vetsinou se react komponenty deklaruji pomoci arrow funkci a tam specifikujes i typ te arrow funkce a co vraci. Tvoje `function` deklarace ted nic nerika o tom, co za navratovy typ ma.



```
import React from "react";
import "./styles.css";

export type Props = {
  name: string;
};

export const Demo: React.FC<Props> = ({ name }) => {
```